jsx/FilterableDataTable.js

```diff
@@ -57,13 +58,17 @@ class FilterableDataTable extends Component {
       />
     );
 
-    const dataTable = (
+    const dataTable = !isNaN(this.props.loading) && this.props.loading < 100 ? (
+      <LoadingBar progress={this.props.loading}/>
+    ) : (
```

Comment:
not in **24.1**

_it seems as though I co-opted the loading prop to serve also a progress prop, if it is represented by a number rather and a boolean._

include as  **CBIG override** and _adapt_ for a push to **main**